skyplane/planner/planner.py

```diff
@@ -277,9 +286,12 @@ def plan(self, jobs: List[TransferJob]) -> TopologyPlan:
             partition_id = jobs.index(job)
 
             # source region gateway program
-            obj_store_read = src_program.add_operator(
-                GatewayReadObjectStore(src_bucket, src_region_tag, self.n_connections), partition_id=partition_id
-            )
+            if isinstance(job, TestCopyJob):
+                obj_store_read = src_program.add_operator(GatewayGenData(64), partition_id=partition_id)
```

Comment:
likely fine for this PR but for the future we may want to have more input on the amount of data to work with for testing, a comment denoting this might be useful